src/color.rs

```diff
@@ -0,0 +1,78 @@
+use crate::zvariant::{self, DeserializeDict, Type};
+use crate::{desktop::screenshot::ColorRequest, Error};
+
+#[derive(DeserializeDict, Clone, Copy, PartialEq, Type)]
+/// The response of a [`ColorRequest`] request.
+///
+/// **Note** the values are normalized.
+#[zvariant(signature = "dict")]
+pub struct Color {
+    color: [f64; 3],
+}
+
+impl Color {
+    /// Red.
+    pub fn red(&self) -> f64 {
+        self.color[0]
+    }
+
+    /// Green.
+    pub fn green(&self) -> f64 {
+        self.color[1]
+    }
+
+    /// Blue.
+    pub fn blue(&self) -> f64 {
+        self.color[2]
+    }
+
+    /// Creates a new builder-pattern struct instance to construct
+    /// [`Color`].
+    ///
+    /// This method returns an instance of [`ColorRequest`].
+    pub fn request() -> ColorRequest {
```

Comment:
This no longer make much sense now, maybe we should figure out how to make it agnostic between settings and color picker?

Reply:
We don't need it to be agnostic. Maybe we can just add a `impl Color` with this single method in the color picker's file

Reply:
I would keep it in the same file and have a new method that returns the accent color?